### PR TITLE
feat:  update icons for widget library-952

### PR DESCRIPTION
**Pull request by Priyanshu44**
https://github.com/awslabs/iot-application/issues/952

## Overview
[UX] Update icons for widget library

## Verifying Changes
image

### Scene Composer
For `scene-composer` package changes specifically, you can preview the component in the published storybook artifact. To do this, wait for the `Publish Storybook` action to complete below.

- Click on the workflow details
- Select the Summary item on the left
- Download the zip file

To run the storybook build locally, you need a local static web server:

```
npm install -g httpserver
cd <Extracted Zip Directory>
httpserver
```

Then open the website http://localhost:8080 to run the doc site.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Review by mnischay — CHANGES_REQUESTED** [review]
Please use cloudscape design tokens , and give a try to have those ICONS components to import a .svg file instead of the svg being part component body

**Comment by Priyanshu44** [comment]
As discussed before the design approvals are pending for this issue so it moved to on hold.

**Pull request closed without merging**
